cxx_qt::Threading does not work with namespace

Hi. It seems that I cannot use cxx_qt::Threading trait with type in namespace. I tested the following code with v0.6.1 and main (6694de64d64f3b976cb8483b4959ef092a4537c2).

```rust
#[cxx_qt::bridge]
pub mod qobject {
    #[namespace = "TestNameSpace"]
    unsafe extern "RustQt" {
        #[qobject]
        type Test = super::TestRust;
    }

    impl cxx_qt::Threading for Test {}
}

#[derive(Default)]
pub struct TestRust;
```

The error is similar as follows:
```
[build] warning: cxx_qt_threading_rust@0.0.0: build/./cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx_qt_threading_rust-3277739e81370077/out/cxx-qt-gen/src/lib.cxx.cpp:389:29: error: ‘::TestCxxQtThread’ has not been declared; did you mean ‘TestNameSpace::TestCxxQtThread’?
[build] warning: cxx_qt_threading_rust@0.0.0:   389 |     ::rust::IsRelocatable<::TestCxxQtThread>::value,
[build] warning: cxx_qt_threading_rust@0.0.0:       |                             ^~~~~~~~~~~~~~~
[build] warning: cxx_qt_threading_rust@0.0.0:       |                             TestNameSpace::TestCxxQtThread
[build] warning: cxx_qt_threading_rust@0.0.0: In file included from cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx_qt_threading_rust-3277739e81370077/out/cxx-qt-gen/src/lib.cxx.cpp:3:
[build] warning: cxx_qt_threading_rust@0.0.0: build/cxxqt/crates/cxx_qt_threading_rust/include/cxx_qt_threading_rust/src/lib.cxxqt.h:9:7: note: ‘TestNameSpace::TestCxxQtThread’ declared here
[build] warning: cxx_qt_threading_rust@0.0.0:     9 | using TestCxxQtThread = ::rust::cxxqt1::CxxQtThread<Test>;
[build] warning: cxx_qt_threading_rust@0.0.0:       |       ^~~~~~~~~~~~~~~
[build] warning: cxx_qt_threading_rust@0.0.0: build/./cargo/build/x86_64-unknown-linux-gnu/debug/build/cxx_qt_threading_rust-3277739e81370077/out/cxx-qt-gen/src/lib.cxx.cpp:389:44: error: template argument 1 is invalid
[build] warning: cxx_qt_threading_rust@0.0.0:   389 |     ::rust::IsRelocatable<::TestCxxQtThread>::value,
[build] warning: cxx_qt_threading_rust@0.0.0:       |                                            ^
```

From what I can gather, in the generated header, `TestCxxQtThread` is placed in `TestNameSpace` namespace. But in the generated source, `TestCxxQtThread` is being referred from the root namespace.

--- a/crates/cxx-qt-gen/src/generator/rust/threading.rs
+++ b/crates/cxx-qt-gen/src/generator/rust/threading.rs
@@ -47,6 +47,7 @@ pub fn generate(
             .cxx_qt_ffi_method("cxxQtThreadIsDestroyed")
             .into_cxx_parts();
 
+    let cxx_qt_thread_namespace = &namespace_ident.namespace;
     let namespace_internals = &namespace_ident.internal;
     let cxx_qt_thread_ident_type_id_str =
         namespace_combine_ident(&namespace_ident.namespace, cxx_qt_thread_ident);
@@ -63,7 +64,7 @@ pub fn generate(
                     // For now we use a type alias in C++ then use it like a normal type here
                     // <https://github.com/dtolnay/cxx/issues/683>
                     #[doc(hidden)]
-                    // TODO: Generate the correct #[namespace] attribute.
+                    #[namespace = #cxx_qt_thread_namespace]
                     type #cxx_qt_thread_ident = cxx_qt::CxxQtThread<#cpp_struct_ident>;
                     include!("cxx-qt/thread.h");
 
@@ -201,6 +202,7 @@ mod tests {
             quote! {
                 unsafe extern "C++" {
                     #[doc(hidden)]
+                    #[namespace = ""]
                     type MyObjectCxxQtThread = cxx_qt::CxxQtThread<MyObject>;
                     include!("cxx-qt/thread.h");
 

--- a/crates/cxx-qt-gen/test_outputs/invokables.rs
+++ b/crates/cxx-qt-gen/test_outputs/invokables.rs
@@ -111,6 +111,7 @@ mod ffi {
     }
     unsafe extern "C++" {
         #[doc(hidden)]
+        #[namespace = "cxx_qt::my_object"]
         type MyObjectCxxQtThread = cxx_qt::CxxQtThread<MyObject>;
         include!("cxx-qt/thread.h");
         #[doc(hidden)]

